### PR TITLE
variable: change "ddl_slow_threshold"

### PR DESCRIPTION
**Pull request by xiekeyi98**
Except changing "ddl_slow"threshold" by POST , we also can use SQL to
change it now.

### What problem does this PR solve? 

HTTP POST is hard to use.
According to this PR , we can use SQL to change "ddl_slow_threshold".

### What is changed and how it works?

Change the variable package so that we can use SQL to change.

```sql
mysql> show variables like "%ddl_slow_%";
+--------------------+-------+
| Variable_name      | Value |
+--------------------+-------+
| ddl_slow_threshold | 300   |
+--------------------+-------+
1 row in set (0.01 sec)

mysql> set session ddl_slow_threshold=12345;
Query OK, 0 rows affected (0.00 sec)

mysql> show variables like "%ddl_slow_%";
+--------------------+-------+
| Variable_name      | Value |
+--------------------+-------+
| ddl_slow_threshold | 12345 |
+--------------------+-------+
1 row in set (0.01 sec)

mysql> set @@session.ddl_slow_threshold=54321;
Query OK, 0 rows affected (0.00 sec)

mysql> select @@session.ddl_slow_threshold;
+------------------------------+
| @@session.ddl_slow_threshold |
+------------------------------+
| 54321                        |
+------------------------------+
1 row in set (0.01 sec)

mysql> 

```

### CODE REVIEW

**Comment by xiekeyi98** [comment]
PTAL @winkyao

File: executor/set_test.go [hunk 1]
```diff
@@ -137,6 +138,15 @@ func (s *testSuite2) TestSetVar(c *C) {
 	tk.MustExec("set character_set_results = NULL")
 	tk.MustQuery("select @@character_set_results").Check(testkit.Rows(""))
 
+	tk.MustExec("set @@session.ddl_slow_threshold=12345")
+	tk.MustQuery("select @@session.ddl_slow_threshold").Check(testkit.Rows("12345"))
+	c.Assert(strconv.FormatUint(uint64(variable.DDLSlowOprThreshold), 10),
```
Comment (winkyao) [inline executor/set_test.go]: `c.Assert(variable.DDLSlowOprThreshold,Equals, uint32(12345))`

File: executor/set_test.go [hunk 2]
```diff
@@ -137,6 +138,15 @@ func (s *testSuite2) TestSetVar(c *C) {
...
+		Equals, "12345")
+	tk.MustExec("set session ddl_slow_threshold=\"54321\"")
+	tk.MustQuery("show variables like 'ddl_slow_threshold'").Check(testkit.Rows("ddl_slow_threshold 54321"))
+	c.Assert(strconv.FormatUint(uint64(variable.DDLSlowOprThreshold), 10),
```
Comment (winkyao) [inline executor/set_test.go]: ditto

File: executor/set_test.go [hunk 3]
```diff
@@ -137,6 +137,13 @@ func (s *testSuite2) TestSetVar(c *C) {
 	tk.MustExec("set character_set_results = NULL")
 	tk.MustQuery("select @@character_set_results").Check(testkit.Rows(""))
 
+	tk.MustExec("set @@session.ddl_slow_threshold=12345")
+	tk.MustQuery("select @@session.ddl_slow_threshold").Check(testkit.Rows("12345"))
+	c.Assert(variable.DDLSlowOprThreshold, 10, uint32(12345))
+	tk.MustExec("set session ddl_slow_threshold=\"54321\"")
+	tk.MustQuery("show variables like 'ddl_slow_threshold'").Check(testkit.Rows("ddl_slow_threshold 54321"))
+	c.Assert(variable.DDLSlowOprThreshold, 10, uint32(54321))
```
Comment (winkyao) [inline executor/set_test.go]: should be `c.Assert(variable.DDLSlowOprThreshold, EQUALS, uint32(54321))`, Please take a look at https://labix.org/gocheck doc.

Comment (winkyao) [reply executor/set_test.go]: The CI failed because of this.

Comment (xiekeyi98) [reply executor/set_test.go]: I am sorry for that.
I will fix it asap.

**Pull request merged**